site/index.ejs

```diff
@@ -39,8 +39,90 @@
         ga('create', 'UA-16654919-1', 'auto');
         ga('send', 'pageview');
     </script><% } %>
+    <style>
+      .accessibilityDay {
+        filter: grayscale(100%);
+        -webkit-filter: grayscale(100%);
+        -moz-filter: grayscale(100%);
+        -ms-filter: grayscale(100%);
+        -o-filter: grayscale(100%);
+      }
+
+      .accessibilityBar {
+        width: 100%;
+        padding: 20px;
+        background-color: #22d59b;
+        text-align: center;
+      }
+
+      .accessibilityBar button {
```

Comment:
Still has default chrome on click. Small but doesn't match behaviour of the other buttons:

![screen shot 2017-05-17 at 12 10 20](https://cloud.githubusercontent.com/assets/4739372/26151267/eec41614-3af9-11e7-8d47-d0b34e3d1195.png)